locales/ja/puppet.po

```diff
@@ -4,24 +4,25 @@
 # FIRST AUTHOR <EMAIL@ADDRESS>, 2018.
 # 
 # Translators:
+# Eriko Kashiwagi <eriko.55@d5.dion.ne.jp>, 2018
+# sayaka inagaki <sayaka-inagaki@to-in.co.jp>, 2018
 # LarissaLane <larissa@puppet.com>, 2018
 # satoko inoue <satoko-inoue@to-in.co.jp>, 2018
-# James Shen <James.Shen@puppet.com>, 2018
-# sayaka inagaki <sayaka-inagaki@to-in.co.jp>, 2018
 # Hidekatsu Tsuru <hidekatsu-tsuru@to-in.co.jp>, 2018
-# Eriko Kashiwagi <eriko.55@d5.dion.ne.jp>, 2018
-# Kojima Ai <ai-kojima@to-in.co.jp>, 2018
 # 梅田智世 <c-umeda@nifty.com>, 2018
+# Kojima Ai <ai-kojima@to-in.co.jp>, 2018
+# Erwin Hom <erwin.hom@puppet.com>, 2018
+# James Shen <James.Shen@puppet.com>, 2018
 # 
 #, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: Puppet automation framework 6.0.4-147-ge097be0\n"
+"Project-Id-Version: Puppet automation framework 5.5.8-108-g9e04bef\n"
```

Comment:
Similar comment as in https://github.com/puppetlabs/puppet/pull/7323/files#r246210869. It looks like the new translations for 6.0.x and master are based off source code from 5.5.x branch `9e04bef`.